src/modules/task/Task.js

```diff
@@ -38,13 +37,25 @@ const useStyles = makeStyles((theme) => ({
     zIndex: 1,
     transition: "ease .3s",
     "& p": {
-      willChange: "color",
+      willChange: "color transform",
       transition: "color .35s",
       color: "gold",
       margin: 0,
       marginRight: ".4em",
     },
   },
+  pulse: {
+    animation: "$pulse infinite alternate .75s ease",
+    color: "lightgreen !important",
+  },
+  "@keyframes pulse": {
```

Comment:
ooooh yeah, gotta love a good pulse effect

Reply:
Love that!